doc/4-Merge.md

```diff
@@ -0,0 +1,349 @@
+# OpenData Merge
+
+This kind of analyzers are based on the basic SQL analyzer, but mastering the latter is not required. The merge framework aim to simplify this task.
+
+
+## Generals concepts
+
+The global idea is to:
+* fetch an OpenData set,
+* load it into database,
+* convert OpenData attributes to OSM tags,
+* compare with OSM by location distance and tags (aka conflate)
+* report differences: missing objects in OSM, objects present in OSM but not in OpenData set, matching objects with different tags or values and more.
+
+
+## General implementation
+
+We aim here to have configuration over code. Nevertheless this configuration is written in Python and may contain Python code when necessary (but not on typical use case). The goal is to easy the process of creating and maintaining this kind of merger analyzers.
+
+The configuration can be seen as a chain:
+- Content is fetched from a `Source`: remote or local file or archive.
+- File format reader: the `Parser` with implementation available for multiple formats like CSV, JSON, GeoJSON, GTFS, SHP... It imports the data into the database.
+- The reader make data available to `Load`. Thos one pre-process and filters data to a convenient schema.
+- The `Conflate` allows to compare and report from
+  - the OSM data filtered by `Select`,
+  - the OpenData converted to OSM tags with `Mapping`
+
+The full merge framework code is in `Analyser_Merge.py`.
+
+The support of the diff mode is not implemented.
+
+
+## Classes of issues
+
+Like other analyzer merge must define classes, but not using `def_class`. Dedicated helpers available:
```

Comment:
"but not using `def_class`" n'est pas très clair. Est-ce que ça serait plus clair comme ça ?

"Like other analyzer merge must define classes, picked from on of these available helpers:"

Reply:
ok